test/integration/smoke/test_vm_life_cycle.py

```diff
@@ -895,11 +895,39 @@ def test_12_start_vm_multiple_volumes_allocated(self):
             startvm=False
         )
 
+        hosts = Host.list(
+            self.apiclient,
+            zoneid=self.zone.id,
+            type='Routing',
+            hypervisor=self.hypervisor,
+            state='Up')
+
+        if self.hypervisor.lower() in ["simulator"]:
+            hypervisor_version = "default"
+        else:
+            res = self.dbclient.execute("select hypervisor_version from host where uuid = '%s'" % hosts[0].id)
```

Comment:
There is no need for this query. The hypervisor version is a response parameter of list hosts

Reply:
Done, thanks